src/javascripts/components/teachers/teachers.scss

```diff
@@ -0,0 +1,14 @@
+.teacher-form {
+  font-family: 'Rye', cursive;
+  color: #f2efe8;
+}
+
+.card-img-top {
```

Comment:
Instead of applying styles to the default bootstrap class, you'll probably want to apply this to your own custom class, so as not to affect other cards that use this class if they don't want to.